__mocks__/react-relay.js

```diff
@@ -1,12 +0,0 @@
-// Comes from https://github.com/facebook/relay/issues/161
-// Should be used in tests as a replacement for `react-storybooks-relay-container`
-//
-var Relay = require.requireActual("react-relay")
-
-module.exports = {
-  graphql: Relay.graphql,
-  QueryRenderer: Relay.QueryRenderer,
-  createFragmentContainer: component => component,
-  createPaginationContainer: component => component,
-  createRefetchContainer: component => component,
-}
```

Comment:
This one also needed to be removed for Relay to actually work, will add to your TODO list.